test/functional/core/job_spec.lua

```diff
@@ -165,8 +165,15 @@ describe('jobs', function()
     else
       nvim('command', "let j = jobstart(['cat', '"..filename.."'], g:job_opts)")
     end
-    eq({'notification', 'stdout', {0, {'abc\ndef', ''}}}, next_msg())
-    eq({'notification', 'stdout', {0, {''}}}, next_msg())
+    expect_msg_seq(
+      { {'notification', 'stdout', {0, {'abc\ndef', ''} } },
+        {'notification', 'stdout', {0, {''} } }
+      },
+      -- Alternative sequence:
+      { {'notification', 'stdout', {0, {'abc\ndef'} } },
+        {'notification', 'stdout', {0, {'', ''} } }
+      }
```

Comment:
Technically we should also expect a final `{'notification', 'stdout', {0, {''} } }`, so in the alternative case it would have 3 chunks (I forgot to do this in one of the cases in my PR):

```
+      -- Alternative sequence:
+      { {'notification', 'stdout', {0, {'abc\ndef'} } },
+        {'notification', 'stdout', {0, {'', ''} } }
+        {'notification', 'stdout', {0, {''} } }
+      }
```